DeviceClassifier/OneLayer/test_OneLayer.py

```diff
@@ -111,35 +58,29 @@ def calc_f1(results, logger, ignore_unknown=False):
 
     # Get the true label assignments
     logger.info('Getting label assignments')
-    with open('opts/label_assignments.json') as handle:
-        label_assignments = json.load(handle)
+    label_assignments = utils.get_labels(args.labels, model_labels=model.labels)
+    if not label_assignments:
+        logger.warn('Could not read label assignments; continuing anyway.')
```

Comment:
will things actually work if it continues anyway?

Reply:
If the label_assignments doesn't return anything useful, the pcaps should theoretically be labeled as unknowns